lambda/create/main.go

```diff
@@ -92,6 +100,14 @@ func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequ
 
 	// save
 	err = l.store.Put(ctx, data)
+	if err != nil {
+		l.logger.Print(err)
+		return shared.ProblemInternalServerError.Respond()
+	}
+
+	// save to static storage as JSON
+	objectKey := fmt.Sprintf("%s/donor-executed-lpa.json", data.Uid)
+	_, err = l.staticLpaStorage.Put(objectKey, data)
```

Comment:
if you are never going to need the PutObjectOutput from this it may be better to make this `Put(...) error`